scripts/deprecate-old-prerelease-versions.ts

```diff
@@ -0,0 +1,38 @@
+import execa = require('execa');
+import { createLogger } from '@sap-cloud-sdk/util';
+
+const logger = createLogger('scripts');
+
+const packageName = '@sap-cloud-sdk/core';
+// const packageName ='@sap-cloud-sdk/util'
+// const packageName ='@sap-cloud-sdk/analytics'
+// const packageName ='@sap-cloud-sdk/test-util'
+// const packageName ='@sap-cloud-sdk/generator'
+
+const dryRun = true;
+
+async function deprecateVersions() {
```

Comment:
I guess this is a convenient tool for clean up?

Reply:
yes I would like to use this to deprecate the old prerelease versions.